### DISCUSSION TITLE
supabase/ssr add useJWT, useSession, useUser hooks for react

### DISCUSSION THREAD

**Discussion by bombillazo** [supabase/supabase#20918] [Feature Requests]
Hey, the old auth-helpers for react had convenient hooks like useUser and useSession to get the data in any component. This feature is missing from supabase/ssr and means I need to keep both libraries around.

It would also be nice to have a direct hook that returns the current JWT.

**Comment by rusakovic** [comment]
confirm. How to use 'use client' if you recommended way for Auth is SSR. Docs say nothing about it
